docs/docs/workflows/troubleshooting.md

```diff
@@ -35,5 +35,10 @@ Constellation uses the default bucket to store logs. Its [default retention peri
 
 :::
 
+</tabItem>
+<tabItem value="aws" label="AWS">
+
+TODO: Is this already supported? Test or write note here!
```

Comment:
No

Reply:
Was implemented, added missing permissions in #461 .